src/components/panels/History/HistoryListEntryJob.vue

```diff
@@ -331,5 +314,18 @@ export default class HistoryListPanel extends Mixins(BaseMixin) {
                 return value
         }
     }
+
+    createThumbnailUrl(thumbnail: FileStateFileThumbnail | undefined) {
```

Comment:
```suggestion
    createThumbnailUrl(thumbnail: FileStateFileThumbnail) {
```

just because of the suggest above.

Reply:
Changes applies, good call :+1: